### PR TITLE
DIV-4629 solicitor details

### PR DESCRIPTION
**Pull request by alexwakeman**
[AOS: Respondent can nominate their solicitor on the RFE.](https://tools.hmcts.net/jira/browse/DIV-4629)

Includes unit and e2e tests

### CODE REVIEW

File: test/.eslintrc.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "no-invalid-this":"off",
     "new-cap": "off",
     "max-len": ["error", {
-        "code": 100
+        "code": 160
```
Comment (alexwakeman) [inline test/.eslintrc.json]: 100 is way too short for my liking

File: test/unit/steps/respondent/reviewApplication.test.js [hunk 2]
```diff
@@ -67,6 +70,9 @@ describe(modulePath, () => {
         reasonForDivorceClaimingAdultery: false
       }
     };
+    if (parseBool(config.features.respSolicitorDetails)) {
```
Comment (qzhou-hmcts) [inline test/unit/steps/respondent/reviewApplication.test.js]: can we use sinon sambox to mock the property and split this test in two, one   with respSolicitorDetails enabled and other with respSolicitorDetails disable?

Comment (alexwakeman) [reply test/unit/steps/respondent/reviewApplication.test.js]: yes good suggestion thanks

File: test/.eslintrc.json (hunk 1 above)
Comment (qzhou-hmcts) [reply test/.eslintrc.json]: Minor: In BE projects we are using 150, maybe we can standardize this value

Comment (alexwakeman) [reply test/.eslintrc.json]: Standardise to 160? Sure! Need to update back-end though 🤣

**Pull request closed without merging**